src/release/issue-triaging.md

```diff
@@ -54,6 +54,16 @@ To see a list of all labels, check out the "labels" page next to the search bar
 [IRLO]: https://internals.rust-lang.org/
 [Discord]: https://discord.gg/rust-lang
 
+### ICE Triage
+For [Issues that have both I-ICE and needs-triage](https://github.com/rust-lang/rust/issues?q=is%3Aissue%20state%3Aopen%20label%3AI-ICE%20label%3Aneeds-triage)
+* If it an older (like latest stable) version of rust, ask for or check the latest nightly (potentially check for duplicates)
+* If it does not have a reproduction, comment asking for one and add S-needs-repro. if there isn't one for around a month it should generally be closed.
+* If the reproduction is not minimal, add `E-needs-mcve` or create a Minimal Complete and Verifyable Example yourself.
+* Add `A-*` labels based on the code that causes the issue (check backtraces!),
+  and the nature of the repro (eg. if the repro is a weird trait impl or the backtrace points to `rustc_trait_selection`, add `A-traits`)
+* Add `F-*`, `requires-*`, and `regression-*` labels as appropriate.
```

Comment:
Remark: see above for more specific advice on those labels.

Reply:
i'm confused as to why you left this comment telling me to look at the other comment on this same line of code.

Reply:
ah, it's not the same line of code, GitHub is just being very confusing.